documentation/modules/ref-example-address-space-output.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// assembly-managing-address-spaces.adoc
+
+[id='ref-example-address-space-output-{context}']
+= Example address space status output
+
+The `AddressSpace` resource contains a `status` field that can be used to retrieve information about
+its state and endpoints. The following output is an example of the output you can get from running
+ifeval::["{cmdcli}" == "kubectl"]
```

Comment:
Is this necessary? Can't we just replace kubectl and oc in the code snippets with {cmdcli}